### ISSUE TITLE
[Guidance] multiple image edit  using Qwen image 2.1

### ISSUE THREAD

**Issue by ak-sangal** [huggingface/optimum-intel#2023]
is it possible to edit multiple images using qwen image 2.1 as its original nature if yes plzz provide script and guidance as i am using intel irisx iGPU with16 GB RAM.by using below script we only edit one image

import torch
from diffusers.utils import load_image
from optimum.intel import OVQwenImage21Img2ImgPipeline

MODEL_DIR = "qwen-image-2.1-ov-int4"
SEED = 21

pipe = OVQwenImage21Img2ImgPipeline.from_pretrained(MODEL_DIR)
pipe.to("GPU")

init_image = load_image("qwen_image21_ov_t2i.png")

prompt = "Add a red party hat on the cat"
image = pipe(
    prompt=prompt,
    image=init_image,
    negative_prompt=" ",
    true_cfg_scale=1.0,
    num_inference_steps=20,
    height=512,
    width=512,
    output_resolution=512,
    generator=torch.Generator("cpu").manual_seed(SEED),
).images[0]
image.save("qwen_image21_ov_edit.png")
print("saved qwen_image21_ov_edit.png", image.size)
